integration-tests/src/test/java/com/redhat/parodos/integration/notification/NotificationRecord.java

```diff
@@ -40,70 +42,170 @@ public void setUp() throws IOException {
 	@Test
 	public void notificationServiceHappyHappyFlow() {
 		String testName = "notificationServiceHappyHappyFlow";
-		logTestStep(testName, "Create the notification messages");
-		createNotificationMessage("type0", "body0", "subject0", "test");
-		createNotificationMessage("type1", "body1", "subject1", "test");
-		createNotificationMessage("type2", "body2", "subject2", "test");
-		createNotificationMessage("type3", "body3", "subject3", "test2");
 
-		validateNotificationRecordCount(testName, 3);
 		assertDoesNotThrow(() -> {
-			PageNotificationRecordResponseDTO list = validateNotificationRecordList(testName, 3);
-			NotificationRecordResponseDTO notificationRecord = list.getContent().get(0);
+			createNotificationMessage(testName, "type0", "body0", "subject0", List.of("test"));
+			createNotificationMessage(testName, "type1", "body1", "subject1", List.of("test"));
+			createNotificationMessage(testName, "type2", "body2", "subject2", List.of("test"));
+			createNotificationMessage(testName, "type3", "body3", "subject3", List.of("test2"));
+
+			countNotificationRecord(testName, 3);
+			PageNotificationRecordResponseDTO list1 = listNotificationRecord(testName, 3);
```

Comment:
is possible to improve the name of object `list1`? To make the test easier to understand

Reply:
Changed